src/Simple/JSON.purs

```diff
@@ -190,14 +192,22 @@ instance readNullable :: ReadForeign a => ReadForeign (Nullable a) where
         TypeMismatch inner other -> TypeMismatch ("Nullable " <> inner) other
         _ -> error
 
-instance readMap :: ReadForeign a => ReadForeign (Map String a) where
+instance readMapNewtype :: ReadForeign a => ReadForeign (Map b a) where
```

Comment:
This seems to be misnamed and dodgy, looks like the newtype version is parked below